google/datalab/contrib/pipeline/_pipeline.py

```diff
@@ -41,16 +41,17 @@ class Pipeline(object):
 from airflow.contrib.operators.bigquery_table_delete_operator import BigQueryTableDeleteOperator
 from airflow.contrib.operators.bigquery_to_bigquery import BigQueryToBigQueryOperator
 from airflow.contrib.operators.bigquery_to_gcs import BigQueryToCloudStorageOperator
-from datetime import datetime, timedelta
+from datetime import timedelta
+from pytz import timezone
 
 """
 
   _default_args_format = """
```

Comment:
Consider giving this a more descriptive name that makes it clear that this is generating a set of standard Airflow attributes. Or, less preferable, add a comment.

Reply:
Done; refactored and renamed to airflow_default_args.